packages/block-editor/src/components/block-settings-menu/block-mode-toggle.js

```diff
@@ -34,9 +34,9 @@ export function BlockModeToggle( { blockType, mode, onToggleMode, small = false,
 
 export default compose( [
 	withSelect( ( select, { clientId } ) => {
-		const { getBlock, getBlockMode } = select( 'core/block-editor' );
+		const { getBlock, getBlockMode, getEditorSettings } = select( 'core/block-editor' );
```

Comment:
This should be `getSettings` if I recall properly. Also, do we want to document that new setting in the code editor default settings?